lib/storage/src/content_manager/toc/collection_meta_ops.rs

```diff
@@ -58,7 +59,13 @@ impl TableOfContent {
             }
             CollectionMetaOperations::DeleteCollection(operation) => {
                 log::info!("Deleting collection {}", operation.0);
-                self.delete_collection(&operation.0).await
+                let res = self.delete_collection(&operation.0).await;
+
+                // We can solve all issues related to this collection
+                if *res.as_ref().unwrap_or(&false) {
+                    issues::solve_by_filter(|code| code.starts_with(&operation.0));
+                }
```

Comment:
Wouldn't it be better to just store the collection name in the issue, and match that directly.

I feel like matching part of a string here may be very error prone here.

For example, I could name my collection: `something.UNINDEXED_FIELD`

Reply:
I guess I didn't check our validation for collection names, so maybe we can use `/` as separator instead?